projects.yml

```diff
@@ -25,60 +24,42 @@
 
 - name: mocha
   repo: https://github.com/mochajs/mocha
-  commit: 09ce746aa925d35317f2624fd36c77a31bb68e24
+  commit: 883ae4b5e1aacd8cf30694da33b391ce58f4cca8
   args:
     - .
     - bin/*
-    - --rule=indent:off
-    - --rule=no-multi-spaces:[error,{ignoreEOLComments:true}]
 
 - name: node
   repo: https://github.com/nodejs/node
-  commit: 0863a0e528ba9ee01f960dce7ffebe8fff3a774c
+  commit: 7babffbe6054754b733098c52786a23df4049a74
   args:
     - --rulesdir=tools/eslint-rules
     - benchmark
     - lib
     - test
     - tools
-    - --rule=indent:off
-    - --rule=no-multi-spaces:[error, {ignoreEOLComments:true}]
-    - --rule=timer-arguments:off
-    - --rule=no-self-assign:[error, {props:false}]
 
 - name: redux
   repo: https://github.com/reactjs/redux
-  commit: ac7cd32a5270eb374611bf73d072e84a1e7567c2
+  commit: 9819b5ef2909cacda0f09c8d29fde229bd7da5fa
   args:
+    - --ext .js,.ts
     - src
     - test
-    - examples
 
-- name: vue
-  repo: https://github.com/vuejs/vue
-  commit: d982e344b39391fe91b6dd91d51b2f0310a45e77
-  args:
-    - src
-    - build
-    - test
-    - --rule=indent:off
-    - --rule=no-multi-spaces:[error,{ignoreEOLComments:true}]
-    - --rule=camelcase:off # There are several `camelcase` errors in the vue codebase for some reason
+# - name: vue
```

Comment:
Waiting on  https://github.com/vuejs/vue/pull/11099 to land before we can merge this/

Reply:
I decided to disable the failing rules and we can update later on.